### DISCUSSION TITLE
Filter by date stopped for rows after 10th August

### DISCUSSION THREAD

**Discussion by sangroxx** [Budibase/budibase#7495] [Help]
Hi!
I have created a simple order tracking app, which has been working fine except for a few minor issues here and there.
However, since last week the users have reported a problem where on the Orders Screen, in the table, the filter by Date column does not fetch any rows on or after 10th August 2022.

No changes were made to the app.

Please checkout the screen recording for more information - 
[Screenrecording](https://app.usebubbles.com/vTBCg4u4YJGdetbb9qQaYf/budibase-ajrl)

**Comment by shogunpurple** [comment]
Hi @sangroxx - sorry to hear you are having issues. Could you possibly send an export of your application and a screenshot of your date column settings in the data UI, so we can debug further? Thank you

**Reply by sangroxx** [reply]
Removed my reply since there was no reply anyway, and my follow-up response seems to be removed. Had the exported app with all the data and the required screenshot in the original reply.
